packages/elements/src/v2/src/components/image-cropper/__styles__/styles.ts

```diff
@@ -0,0 +1,307 @@
+/* eslint-disable max-len */
+import { css } from 'linaria'
+
+// CSS from react-image-cropper
+export const reactImageCropGlobalStyles = css`
```

Comment:
Just a question but is it possible to just do an @import from the node module into this file?

Reply:
So we get the updates when the package updates

Reply:
I used to do that in `<Spreadsheet />` and the CSS gets stripped out after build, but I'm not sure if we can do that with linaria in elements v2. Let's me try and I'll get back to you
Thanks 👍